src/backend/looped/self/toggle_passive.cpp

```diff
@@ -21,15 +22,15 @@ namespace big
 				g_notification_service->push_warning("Disabled passive mode", "Disabled passive mode because you started mission / joined CEO/MC");
 				return;
 			}
-			*scr_globals::tuneables.at(27371).as<int*>() = 0; // End Passive Time = 0s
-			*script_global(1574582).as<bool*>() = true;
+			*g_tunables_service->get_tunable<int*>(-29732167) = 0; // End Passive Time = 0s
+			*scr_globals::passive.as<PBOOL>()                 = TRUE;
 		}
 
 		virtual void on_disable() override
 		{
-			*script_global(1574582).as<bool*>() = false;
+			*scr_globals::passive.as<PBOOL>() = FALSE;
 			NETWORK::SET_LOCAL_PLAYER_AS_GHOST(false, false);
-			*scr_globals::tuneables.at(27371).as<int*>() = 30000;
+			*g_tunables_service->get_tunable<int*>(-29732167) = 30000;
```

Comment:
The fuck is this `-29732167`? Any way on finding it again if it changes?

Reply:
It’s just the JOAAT’s integer representation. It’s what was in the tunables file. I tried looking for JOAATs that matched it, but all I could find were collisions. The tunable service will auto update it with the matching JOAAT value.